File: pset6/sentimental/mario/less/mario.py
from cs50 import get_int

while True:
    n = get_int("Height = ")
    if n >= 0 and n < 24:
        break

for i in range(n):
    for s in range(n - i - 1):
        print(" ", end="")
    for h in range(i + 2):
        print("#", end="")
    print()

File: pset1/mario/more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    {

        do
        {
            n = get_int("Height = ");
        }
        while ((n < 0) || (n > 23));

        for (int i = 0; i < n ; i++)
        {
            for (int s = 1; s < (n - i); s++)
            {
                printf(" ");
            }
            for (int h1 = 1; h1 <= (i + 1); h1++)
            {
                printf("#");
            }
            printf("  ");
            for (int h2 = 1; h2 <= (i + 1); h2++)
            {
                printf("#");
            }
            printf("\n");
        }
        return 0;
    }
}
